tests/providers/amazon/aws/hooks/test_glue_crawler.py

```diff
@@ -109,16 +113,87 @@ class MockException(Exception):
         self.assertEqual(response, True)
         mock_get_conn.return_value.get_crawler.assert_called_once_with(Name=mock_crawler_name)
         mock_get_conn.return_value.update_crawler.assert_called_once_with(**mock_config_two)
 
+    @mock.patch.object(GlueCrawlerHook, "get_conn")
+    @mock.patch.object(StsHook, "get_conn")
+    @mock.patch.object(StsHook, "get_account_number")
+    def test_update_tags_not_needed(self, mock_get_account_number, mock_sts_conn, mock_get_conn):
+        mock_get_conn.return_value.get_crawler.return_value = {"Crawler": mock_config}
+        mock_get_conn.return_value.get_tags.return_value = {"Tags": mock_config["Tags"]}
+        mock_get_account_number.return_value = AWS_ACCOUNT_ID
+        crawler_arn = f"arn:aws:glue:{AWS_REGION}:{AWS_ACCOUNT_ID}:crawler/{mock_crawler_name}"
+
+        response = self.hook.update_tags(mock_crawler_name, mock_config["Tags"])
+        self.assertEqual(response, False)
+        mock_get_conn.return_value.get_tags.assert_called_once_with(ResourceArn=crawler_arn)
+        mock_get_conn.return_value.tag_resource.assert_not_called()
+        mock_get_conn.return_value.untag_resource.assert_not_called()
+
+    @mock.patch.object(GlueCrawlerHook, "get_conn")
+    @mock.patch.object(StsHook, "get_conn")
+    @mock.patch.object(StsHook, "get_account_number")
+    def test_remove_all_tags(self, mock_get_account_number, mock_sts_conn, mock_get_conn):
+        mock_get_conn.return_value.get_crawler.return_value = {"Crawler": mock_config}
+        mock_get_conn.return_value.get_tags.return_value = {"Tags": mock_config["Tags"]}
+        mock_get_account_number.return_value = AWS_ACCOUNT_ID
+        crawler_arn = f"arn:aws:glue:{AWS_REGION}:{AWS_ACCOUNT_ID}:crawler/{mock_crawler_name}"
+
+        response = self.hook.update_tags(mock_crawler_name, {})
+        self.assertEqual(response, True)
+        mock_get_conn.return_value.get_tags.assert_called_once_with(ResourceArn=crawler_arn)
+        mock_get_conn.return_value.tag_resource.assert_not_called()
+        mock_get_conn.return_value.untag_resource.assert_called_once_with(
+            ResourceArn=crawler_arn, TagsToRemove=["test", "bar"]
+        )
+
+    @mock.patch.object(GlueCrawlerHook, "get_conn")
+    @mock.patch.object(StsHook, "get_conn")
+    @mock.patch.object(StsHook, "get_account_number")
+    def test_replace_tag(self, mock_get_account_number, mock_sts_conn, mock_get_conn):
+        mock_get_conn.return_value.get_crawler.return_value = {"Crawler": mock_config}
+        mock_get_conn.return_value.get_tags.return_value = {"Tags": mock_config["Tags"]}
+        mock_get_account_number.return_value = AWS_ACCOUNT_ID
+        crawler_arn = f"arn:aws:glue:{AWS_REGION}:{AWS_ACCOUNT_ID}:crawler/{mock_crawler_name}"
+
+        mock_config_two = deepcopy(mock_config)
+        mock_config_two.pop("Tags")
+        response = self.hook.update_tags(mock_crawler_name, {"test": "bla", "bar": "test"})
+        self.assertEqual(response, True)
```

Comment:
instead of `TestCase.assert*`methods use regular `assert`

```suggestion
        assert response
```